### DISCUSSION TITLE
For one client, folder names missing first character in virtual folder

### DISCUSSION THREAD

**Discussion by dwgreen1** [drakkan/sftpgo#1780] [Q&A]
I have a virtual folder (Azure Blob backend) set up for some clients. For most every SFTP client I have tested, it works just fine, but I have one user who reports that the first character is missing from folder names, which prevents them from traversing the directories in the usual manner. So instead of something like this:

```text
2024 Records/
2023 Records/
2022 Records/
```

It appears like this:

```text
024 Records/
023 Records/
022 Records/
```

I did not think this was necessarily an SFTPGo issue, but if I swap out the virtual folder with just a local filesystem, then the folder structure _does_ appear normally. I have not been able to figure out what might be the issue on the client side. 

I am curious if anyone else in the community has experienced something similar?

**Comment by drakkan** [comment]
Interesting, this is the first time we have received a report like this. It must be an edge case with this particular client (or some weird bug in the client itself).
As a community user feel free to investigate and submit a PR. Thanks.

**Comment by dwgreen1** [comment]
For what it's worth, I did confirm that the problem was on the client's side. Their software seemed to trip up on the 1970 epoch date on Azure blob "folders", that being the date because there is no actual timestamp for these. I wasn't confident that they would fix it any time soon, so I just swapped everything over to an Azure File Share instead and mounted that. Solves the problem.
